src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import './style.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Books from './pages/BOOKS';
+import Categories from './pages/CATEGORIES';
+
+const root = createRoot(document.getElementById('root'));
+root.render(
+  <BrowserRouter>
+    <Routes>
+      <Route path="/bookstore" element={<Navbar />}>
+        <Route index element={<Books />} />
+        <Route path="/bookstore/Categories" element={<Categories />} />
+      </Route>
+    </Routes>
```

Comment:
- [ ] Please make the book routes the default page `( you can access it with / only route )` to not show a blank page when visiting the site for the first time as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Books%20%2D-,the%20default%20view,-%3A).